torch_xla/csrc/runtime/runtime.cc

```diff
@@ -19,10 +20,14 @@ ComputationClient* CreateClient() {
     tsl::testing::InstallStacktraceHandler();
   }
 
+  static bool use_ifrt = sys_util::GetEnvBool("XLA_USE_IFRT", false);
```

Comment:
why not just `IFRT_DEVICE`? Do you intend to hide this under the PJRT env var?

Reply:
We still use `PJRT_DEVICE` for the "IFRT wraps PJRT" case, since we're initializing exactly the same client. This is just a switch to enable or disable the IFRT wrapper.